modules/menu/views/about.php

```diff
@@ -5,7 +5,7 @@
   <table class="popup-about-table">
     <tr class="popup-about-row popup-about-row-links">
       <td>
-        <a href="https://kb.op5.com/x/UYEK">Knowledge Base</a>
+        <a href="https://docs.itrsgroup.com">Knowledge Base</a>
```

Comment:
Shouldn't it be a link to https://support.itrsgroup.com/hc/en-us ?

Reply:
I was under the impression that the above link would possibly change in the future to something more like https://docs.itrsgroup.com/docs/geneos/4.10.0/index.html with version specifics. I find it cleaner to link to the base URL of the docs which will likely not change for a while so that we don't have to update this link in the foreseeable future.

Reply:
Fair enough, I think this looks good!